Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -93,7 +92,7 @@ def main(cls, args, repository, **kwargs):
                 break
 
         if not conflict_pr:
-            sys.stderr.write('No conflict pull request found with branch {}\n'.format(expected_branch))
+            sys.stderr.write(f'No conflict pull requests found for {radar_id}')
```

Comment:
You could make this cleaner by using `radar_obj.link` which will return a string in the form of `rdar://1234`